x/psiphon/psiphon_test.go

```diff
@@ -28,6 +29,18 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func newTestConfig(tb testing.TB) (*DialerConfig, func()) {
```

Comment:
Should it be `testing.T` instead?

```suggestion
func newTestConfig(t *testing.T) (*DialerConfig, func()) {
```